provision/InventoryFilters/GenericInventory.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import abc
+import StringIO
+from ansible import errors
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
+from AWSInventory import AWSInventory
+from OpenstackInventory import OpenstackInventory
+from GCloudInventory import GCloudInventory
+from InventoryFilter import InventoryFilter
+
+
+class GenericInventory(InventoryFilter):
+    def __init__(self):
+        InventoryFilter.__init__(self)
+        self.filter_classes = {
```

Comment:
I continue to question this list of classes. Could it be pulled from the linchpin_config.yml or elsewhere? I'd like to see this be more dynamic. Or possibly, it could be pulled from the classes that have <something>Inventory.py in the python path?

I have code that can do the instantiation of the classes from a config file or the like in a pythonic way, so that shouldn't be too troublesome to implement.